src/widgetresize.js

```diff
@@ -29,57 +30,73 @@ export default class WidgetResize extends Plugin {
 	}
 
 	init() {
-		this.resizers = [];
-		this.activeResizer = null;
+		/**
+		 * A map of resizers created using this plugin instance.
+		 *
+		 * @type {Map.<module:engine/view/containerelement~ContainerElement, module:widget/widgetresize/resizer~Resizer>}
+		 */
+		this.resizers = new Map();
```

Comment:
The fact that you had to change the shape of this object to serve some internal changes indicates that it should be encapsulated. If you need a map, make it protected/private (depending on whether you need it for tests). Add absolutely minimal API, depending on what's needed. E.g., in the other place, you need something like `getResizerByHost()`.

Reply:
It was more natural to fetch by widget view element (as the resizer host might differ across different widget types), so there's `_getResizerByViewElement()`.

As of `resizers` member, I still need access to this map in few `WidgetResize` methods, so it became a private property.